docs/integrations/sigrid-api-documentation.md

```diff
@@ -584,12 +575,65 @@ DELETE https://sigrid-says.com/rest/analysis-results/api/v1/objectives/{customer
 This would delete the system-level test code ratio objective, if it exists at the system level, and 
 return 204. In case the objective does not exist at the system level, the endpoint returns 404. 
 
+#### Retrieving objectives status
+
+    GET https://sigrid-says.com/rest/analysis-results/api/v1/objectives-evaluation/{customer}?startDate={start}&endDate={end}
+    
+This will return the current objectives status and trend for all systems in your portfolio, for the specified time period. The `startDate` and `endDate` parameters accept dates in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601), for example 2024-05-17. 
```

Comment:
I prefer "delta" instead of "trend": the former implies exactly two points in time (the start date and the end date), while the latter implies many points in time (a number of time points between the start date and end date):
```suggestion
This will return the current objectives status and delta for all systems in your portfolio, for the specified time period. The `startDate` and `endDate` parameters accept dates in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601), for example 2024-05-17. 
```